### DISCUSSION TITLE
How to get thre newer versions?

### DISCUSSION THREAD

**Discussion by Gfiti** [alexjustesen/speedtest-tracker#264] [Q&A] [answered]
As the titel says. I'm trying to install it on my Synology NAS, following the Docs instructions.

It installs 0.4.2 every time though. :(

**Comment by mztiq** [comment]
Are you installing with the `docker run` command via Synologys Task Scheduler?  
Post some more details like the commands you're using please.

**Comment by Gfiti** [comment, marked as answer]
Yeah I use a sceduled Task with the code that the doc has for "Docker (Sqlite)"

```
docker run -d --name speedtest-tracker \
    -p 8080:80 \
    -p 8443:443 \
    -e PUID=1000 \ 
    -e PGID=1000 \
    -v /path/to/directory:/config \
    ghcr.io/alexjustesen/speedtest-tracker:latest
    restart: unless-stopped

```

This right here, adjusted for my used directory and UID etc. ofc.

Edit: Had to also delete the image in the docker menu, which I forgot.

**Reply by alexjustesen** [reply]
Lates should always pull the er.... latest. Maybe try deleting the existing image and pull latest again?

**Reply by Gfiti** [reply]
> Lates should always pull the er.... latest. Maybe try deleting the existing image and pull latest again?

Looks like I got it now. I thought deleteing the container and the folder was enough. But I just noticed the Image is still on a seperate menu that I also had to delete. x)

Thank you though, and sorry for wasting yout time! Happy Holidays!

**Reply by alexjustesen** [reply]
No problem at all, Happy Holidays and thanks for using my little side project!
